### DISCUSSION TITLE
Always show button (bulkActions)

### DISCUSSION THREAD

**Discussion by LaMance22** [rappasoft/laravel-livewire-tables#1238] [General]
Hi.

Is it possible to make the "bulkActions" button always visible? after update, it appears only after selecting a row in the table. I understand that this may be the case, but in my case it will be better if it is always visible.

https://rappasoft.com/docs/laravel-livewire-tables/v2/bulk-actions/available-methods

**Comment by lrljoe** [comment]
Please open an issue for this so that I can track it in the project more easily.

If you do that then it should make it into the next version, but not guaranteed, it may be the version after.

You'll need to mark it as a bug, which it is as I've removed a feature in error!

**Reply by lrljoe** [reply]
I've done this on your behalf, and this will be in the next release if all goes smoothly!

**Reply by lrljoe** [reply]
PR #1240 Should have this functionality restored, please feel free to test!
